plugin/coverage-highlight.vim

```diff
@@ -1,15 +1,35 @@
 " File: coverage-highlight.vim
 " Author: Marius Gedminas <marius@gedmin.as>
-" Version: 3.3
-" Last Modified: 2021-02-04
+" Version: 3.3.1
+" Last Modified: 2021-04-02
+" Contributors: Louis Cochen <louis.cochen@protonmail.ch>
```

Comment:
:+1:

Reply:
(I changed the version to 3.4 because I kinda like semver and a new configuration option feels like a feature.)